moulinette/authenticators/ldap.py

```diff
@@ -76,7 +76,7 @@ def is_authenticated(self):
 
     def authenticate(self, password):
         try:
-            con = ldap.initialize(self.uri)
+            con = ldap.ldapobject.ReconnectLDAPObject(self.uri, retry_max=5, retry_delay=0.1)
```

Comment:
0.1 sec for `retry_delay` sounds a bit short ? What about 0.5 ?

Reply:
Well, maybe Yes.

I need to do some more test. I firstly tried with 10 (which is clearly too long). And the test failed because the connection was not re-establish yet.

So this parameter might depends of the processor. If the CPU is really quick it might be not too short.

Reply:
Tested with 3 sec and it's also OK.

I purpose to put `retry_max=10, retry_delay=0.5`